frontend/src/styles/tailwind.css

```
@@ -108,92 +108,1548 @@
-    width: 1px;
-    height: 1px;
-    padding: 0;
-    margin: -1px;
+  .sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)